contrib/datawave-quickstart/bin/query.sh

```diff
@@ -131,21 +131,32 @@ function setQueryIdFromResponse() {
 }
 
 function prettyPrintJson() {
-    local PY=$( which python )
-    if [ -n "${PY}" ] ; then
-        echo "${1}" | ${PY} -c 'from __future__ import print_function;import sys,json;data=json.loads(sys.stdin.read()); print(json.dumps(data, indent=2, sort_keys=True))'
-        local exitStatus=$?
-        echo
-        if [ "${exitStatus}" != "0" ] ; then
-           printRawResponse "${1}"
-           warn "Python encountered error. Printed response without formatting"
-           echo
+    PY_CMD='"from __future__ import print_function; import sys,json; data=json.loads(sys.stdin.read()); print(json.dumps(data, indent=2, sort_keys=True)"'
+        PY3=$(which python3 2>/dev/null)
+        PY2=$(which python2 2>/dev/null)
```

Comment:
I would be interested to see if you could find a way to simplify this a little more. This approach adds some repeating code to handle the two cases. Maybe you could try to consolidate them into one variable, so you do not have to repeat any code.